cloud-controller-manager/do/loadbalancers.go

```diff
@@ -923,7 +927,7 @@ func getProtocol(service *v1.Service) (string, error) {
 	}
 
 	switch protocol {
-	case protocolTCP, protocolHTTP, protocolHTTPS, protocolHTTP2:
+	case protocolTCP, protocolUDP, protocolHTTP, protocolHTTPS, protocolHTTP2:
```

Comment:
`getProtocol` returns the default protocol that is to be used when no explicit protocol was specified for a particular port. (The function would probably be better named `getDefaultProtocol`.) This is needed because we support TCP and various HTTP* flavors.

I'm thinking that we do not need to support this for UDP since selection of the UDP protocol is determined by the protocol value on the Service port definition (i.e., customers need to specify it explicitly anyway). WDYT?